test/conformance/ptp/ptp.go

```diff
@@ -192,7 +188,7 @@ var _ = Describe("[ptp]", func() {
 						ptpMasterRunningPods = append(ptpMasterRunningPods, pod)
 					}
 				}
-				if discovery.Enabled() {
+				if discovery.Enabled() || isSingleNode {
```

Comment:
Why do we need to check if isSingleNode?  If it's a single node, won't discovery always be enabled?

Reply:
same here I assumed that irrespective of the value of discovery enabled, single node means that discovery would always be enabled. will change it